src/igvfd/metadata/decorators.py

```diff
@@ -20,3 +20,19 @@ def wrapper(context, request):
             return func(context, request)
         return wrapper
     return decorator
+
+
+def allowed_types_v2(types):
+    def decorator(func):
+        @wraps(func)
+        def wrapper(context, request):
+            qs = QueryString(request)
+            type_filters = qs.get_type_filters()
```

Comment:
Do we need to check that `len(type_filters) > 0` here?

Reply:
Should return empty list.